Update CSPM regions.md (#11326)
DataDog/documentation: content/en/security_platform/cspm/findings.md

> {{< site-region region="us3,gov,eu" >}}

Reviewer: ```suggestion
{{< site-region region="us3,gov" >}}
```

<div class="alert alert-warning">

> Cloud Security Posture Management is not currently available in US1-FED and US3.

Reviewer: ```suggestion
Cloud Security Posture Management is not available in US1-FED or US3.
```
